SpaceX/Services/TransitionManager.swift

```diff
@@ -22,6 +26,11 @@ protocol Transitionable {
     func copyForView(_ subView: UIView) -> UIView
 }
 
+public enum TransitionType {
```

Comment:
No need for `public`. Change it to `private`

Reply:
Property cannot be declared public because its type uses a private type. Resolved.